vulcanai/tests/models/test_metrics.py

```diff
@@ -294,13 +293,11 @@ def test_get_auc(self, metrics):
 
     def test_run_test(self, metrics, cnn_class):
         """Test that run_test returns values as expected."""
+        warnings.filterwarnings("ignore", category=sklearn.exceptions.UndefinedMetricWarning)
```

Comment:
is there another way to make sure the warning goes away without simply ignoring it? this applies to all the filter warnings?

Reply:
this is just to deal with the sklearn deprecated stuff